blocks/lab/__card-button/lab__card-button.css

```diff
@@ -24,5 +24,12 @@
     }
 }
 
-
-
+@media screen and (min-width: 1125px){
```

Comment:
Isn't it better to add for button `:hover` with `cursor: pointer`?

Reply:
Done